js/btcmarkets.js

```diff
@@ -520,7 +520,7 @@ module.exports = class btcmarkets extends Exchange {
             'baseVolume': baseVolume,
             'quoteVolume': quoteVolume,
             'info': ticker,
-        };
+        });
```

Comment:
Let's propagate `market` as the second argument to safeTicker everywhere.

Reply:
We will need to rewrite lines 481-494 for `safeMarket`. See how that is done in the other exchanges.